.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+registries:
+  rubygems-server-enterprise-contribsys-com:
+    type: rubygems-server
+    url: https://enterprise.contribsys.com/
```

Comment:
Are there docs you followed for this?

Reply:
Dependabot created this automatically too